doc/ref_impl/cntt-ri/chapters/chapter04.md

```diff
@@ -92,3 +92,130 @@ Talk to the Lab Owners if special hardware or topology requirements are needed,
+reliable access to project and release resources. Community labs are provided as a service by companies and are not
+controlled by Pharos however our goal is to provide easy visibility of all lab capabilities and their usage at all-times.
+
+A requirement of Pharos labs is to provide bare-metal for development, deployment and testing. This is resource intensive
+from a hardware and support perspective while providing remote access can also be very challenging due to corporate IT
+policies. Achieving a consistent look and feel of a federated lab infrastructure continues to be an objective. Virtual
+environments are also useful and provided by some labs. Jira is currently used for tracking lab operational issues as
+well as for Pharos project activities.
+
+<a name="4.5.1"></a>
+### 4.5.1 Pharos Specification
+
+The Pharos Specification defines a hardware environment for deployment and testing of the OPNFV platform release.
+
+Pharos lab infrastructure has the following objectives:
+
+* Provides secure, scalable, standard and HA environments for feature development
+* Supports the full Euphrates deployment lifecycle (this requires a bare-metal environment)
+* Supports functional and performance testing of the Euphrates release
+* Provides mechanisms and procedures for secure remote access to Pharos compliant environments for OPNFV community
+* Deploying OpenStack in a Virtualized environment is possible and will be useful, however it does not provide a fully
+  featured deployment and realistic test environment for the Euphrates release of OPNFV.
+
+The high level architecture is outlined in the following diagram:
+
+<img src="../figures/ch04_opnfv_pharos_high_level.jpg" title="OPNFV Pharos High Level Architecture">
+
+A pharos compliant OPNFV test-bed provides:
+
+* One CentOS/Ubuntu jump server (Foundation Node) which can be used to perform the OpenStack RI installation, or host any
+  additional software needed.  This server may also participate in the OpenStack cluster if desired instead of acting as
+  a dedicated services node.
+* 5 target nodes which can be used in any combination, such as:
+  * 3 controller nodes + 2 compute/storage nodes
+  * 1 controller node + 4 compute/storage nodes
+  * 1 controller node + 1 compute node + 3 storage nodes
+* A configured network with the ability to provide the following networks:
+  * Out-of-band Management: Used for access to the lights out (ILO/IMPI/Redfish) network for the purpose of managing the
+    bare metal aspects of the servers, such as power control, BIOS configuration, etc.
+  * External (DMZ): Used to provide VMs with Internet access.  Directly accessible from the VPN.
+  * Provisioning / In-band Management (Admin): to perform management operations on the hypervisor software for each
+    node.  Can also be used for bootstrapping images using PXE or other installation technologies.
+  * API Access (Public): Exposes all OpenStack APIs, including the OpenStack Networking API, to tenants.
```

Comment:
@markbeierl Does "tenants" in this case refer to "OpenStack tenants (projects)?"